### PR TITLE
C++ query module tutorial

### PR DESCRIPTION
**Pull request by antepusic**
New tutorial/developer guide for query modules in C++.

### CODE REVIEW

File: mage/tutorials/create-a-new-module-cpp.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+## Querying
+
+As querying is independent of the query module’s implementation language, all relevant
+information is again available in the Python tutorial.
+
+## Testing
+
+The same remark holds for testing – relevant information is provided inside the Python
+tutorial.
```
Comment (jmatak) [inline mage/tutorials/create-a-new-module-cpp.md:217]: This should not refer to Python tutorial, therefore needs to be extracted: @antoniofilipovic

**Review by jmatak — CHANGES_REQUESTED** [review]

File: mage/tutorials/create-a-new-module-cpp.md (hunk 1 above)
Comment (antoniofilipovic) [reply mage/tutorials/create-a-new-module-cpp.md]: Done!

File: mage/tutorials/create-a-new-module-cpp.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+## Prerequisites
+
+There are three options for installing and working with Memgraph MAGE:
+
+1. **Pulling the `memgraph/memgraph-mage` image**:  check the `Docker Hub` tab in the
+    [installation guide](/mage/installation.md).
+2. **Building a Docker image from the MAGE repository**: check the `Docker build` tab 
+    inside the [installation guide](/mage/installation.md).
+3. **Building MAGE from source**: check the `Build from source on Linux` tab in the
+    [installation guide](/mage/installation.md).
```
[the remark this answers is not in the document]
Comment (antoniofilipovic) [reply mage/tutorials/create-a-new-module-cpp.md]: Fixed!

File: mage/tutorials/create-a-new-module-cpp.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
+--
+id: create-a-new-module-cpp
+title: Developing a query module in C++
+sidebar_label: Create a C++ query module
+---
```
[the remark this answers is not in the document]
Comment (antoniofilipovic) [reply mage/tutorials/create-a-new-module-cpp.md]: Fixed!

**Review by jmatak — APPROVED** [review]

**Pull request merged**